gem/settings/base.py

```diff
@@ -574,8 +574,30 @@ def get_default_template(site_layout_base, site_layout):
     MEDIA_URL = "https://%s/" % AWS_S3_CUSTOM_DOMAIN
     DEFAULT_FILE_STORAGE = 'storages.backends.s3boto.S3BotoStorage'
 
-PWA_SERVICE_WORKER_PATH = join(
-    PROJECT_ROOT, 'templates', SITE_LAYOUT_BASE, 'serviceworker.js')
+PWA_APP_NAME = 'Springster-APP'
```

Comment:
@codiebeulaine these settings are used for the culture brand sites as well right?
maybe we should make them configurable through the settings then?
although is that not possible because the manifest needs to be hard-coded?

Reply:
Ah, I guess this is only loaded in the Springster templates anyway so maybe not neccessary.

Reply:
Yeah initially we only had PWA for springster. And also if you look below the other settings that were there are for sprignster